_posts/2022-12-24-map-my-village-campaign.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: post
+title: OSM Ghana Improves Access To Ghanaian Villages With Mapping Party 
+featured: true
+author: James
```

Comment:
Kindly make a new PR to update, name here should have been lowercase as defined in https://github.com/osmghana/osmgh/blob/923af7837c30e2ee80890677f3a589fdd37c7fc1/_config.yml#L121